src/apollo/index.d.ts

```diff
@@ -30,7 +30,7 @@ export interface SubgraphLiquidityMiningCampaign {
     token0: SubgraphToken
     token1: SubgraphToken
   }
-  liquidityMiningPositions: { id: string; stakedAmount: string }[]
+  liquidityMiningPositions?: { id: string; stakedAmount?: string }[]
 }
 
 export interface SubgraphSingleSidedStakingCampaign {
```

Comment:
These types should be generated by the generator. It's best to remove these

Reply:
Since the graphql schema and codebase types don't quite match, can we merge this fix and tackle the bigger issue later?